### PR TITLE
Profile Page Prototype

### PR DESCRIPTION
**Pull request by jayyduran0108**
I created the ProfileServlet which grabs the username the user clicked on. I create the profile.jsp which verifies if the user is looking at their own profile or not.

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/profile.jsp
Comment (chilimalone) [inline src/main/webapp/WEB-INF/view/profile.jsp]: extra <

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: remove dead code before you submit.

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: add new line before the class declaration

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: add newline before this to separate different library imports

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: add newline before to separate library imports (for readability)

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: I think the existing code all uses 4 spaces for indent. Please use 4 spaces for consistency :)

File: src/main/webapp/WEB-INF/view/profile.jsp
Comment (chilimalone) [inline src/main/webapp/WEB-INF/view/profile.jsp]: edit date?

**Review by chilimalone — CHANGES_REQUESTED** [review]
Good job! Please wait for reviews before merging your code in.  I've just a few nits

File: src/main/webapp/WEB-INF/view/profile.jsp
Comment (jayyduran0108) [reply src/main/webapp/WEB-INF/view/profile.jsp]: done

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/webapp/WEB-INF/view/profile.jsp
Comment (jayyduran0108) [reply src/main/webapp/WEB-INF/view/profile.jsp]: done

**Comment by chilimalone** [comment]
Hey, thanks for responding to my comments! Please push your changes to the branch and let me know so I can confirm that the changes are present and approve the merge :)

**Review by annepham25 — APPROVED** [review]

**Pull request merged**
